.github/workflows/reusable-delivery-issue-chores.yml

```diff
@@ -184,7 +184,7 @@ jobs:
       # Create links in Jira to GitHub Issues
       # We still need to create a Jira User and update the server_host
       - name: Create links to GitHub in Jira issues
-        uses: Fgerthoffert/actions-create-jira-link@v1.0.0
+        uses: Fgerthoffert/actions-create-jira-link@v1.2.0
```

Comment:
is that intended to include this change as part of this PR?

Reply:
Thank you! it was not